plugins/inputs/intel_powerstat/intel_powerstat.go

```diff
@@ -75,21 +80,46 @@ func (p *PowerStat) Init() error {
 	if err != nil {
 		return err
 	}
+
+	p.initMSR()
+	p.initRaplService()
+
+	if !p.areCoreMetricsEnabled() && !p.areGlobalMetricsEnabled() {
+		return fmt.Errorf("all configuration options are empty or invalid. Did not find anything to gather")
+	}
+
+	p.fillCPUBusClock()
+	return nil
+}
+
+func (p *PowerStat) initMSR() {
 	// Initialize MSR service only when there is at least one metric enabled
 	if p.cpuFrequency || p.cpuBusyFrequency || p.cpuTemperature || p.cpuC0StateResidency || p.cpuC1StateResidency ||
-		p.cpuC6StateResidency || p.cpuBusyCycles || p.packageTurboLimit || p.packageUncoreFrequency {
+		p.cpuC6StateResidency || p.cpuBusyCycles || p.packageTurboLimit || p.packageUncoreFrequency || p.packageCPUBaseFrequency {
 		p.msr = newMsrServiceWithFs(p.Log, p.fs)
 	}
+}
+
+func (p *PowerStat) initRaplService() {
 	if p.packageCurrentPowerConsumption || p.packageCurrentDramPowerConsumption || p.packageThermalDesignPower || p.packageTurboLimit ||
-		p.packageUncoreFrequency {
+		p.packageUncoreFrequency || p.packageCPUBaseFrequency {
 		p.rapl = newRaplServiceWithFs(p.Log, p.fs)
 	}
+}
 
-	if !p.areCoreMetricsEnabled() && !p.areGlobalMetricsEnabled() {
-		return fmt.Errorf("all configuration options are empty or invalid. Did not find anything to gather")
-	}
+// fill CPUBusClockValue if required
+func (p *PowerStat) fillCPUBusClock() {
+	if p.packageCPUBaseFrequency {
+		// cpuBusClock is the same for every core/socket.
+		busClockInfo := p.getBusClock("0")
+		if busClockInfo == 0 {
+			p.Log.Debug("Disabling package metric: cpu_base_frequency_mhz. Can't detect bus clock value")
```

Comment:
Shouldn't this be a **warning** here as people enable this feature but won't see it? The debug message might go unnoticed for the average Joe...

Reply:
@srebhan Changed. 

I don't know what's going on with `run-readme-linter` job, but for this readme result is:
`Pass plugins/inputs/intel_powerstat/README.md`